internal/tshelpers/custom_commands.go

```diff
@@ -31,6 +32,8 @@ import (
 // 						   | during the comparison.																   |
 // IsoFileCmpRegEx 		   | checks that file context extracted directly from the ISO matches the content of the   | isoFileCmpRegEx node.x86_64.iso /EFI/redhat/grub.cfg expected/grub.cfg
 //						   | local file, by applying a regex comparison.										   |
+// IsoSizeMin 		   | checks that the ISO is greater than a minimum number of bytes                         | isoSizeMin agent.x86_64.iso 100000000
```

Comment:
Could we specify Min and Max by megabytes? It would be easier to comprehend.

Reply:
Good idea. Changed.